deploy/manifests/dev/us-east-2/cluster/flux-system/cluster-cd.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: GitRepository
+metadata:
+  name: storetheindex
+spec:
+  interval: 5m
+  url: https://github.com/filecoin-project/storetheindex.git
+  ref:
+    # TODO: update to main branch once PR is ready for review.
+    branch: masih/infra-dev-setup
+---
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: cluster
+spec:
+  serviceAccountName: kustomize-controller
+  interval: 5m
+  path: "./deploy/manifests/dev/us-east-2/cluster"
+  sourceRef:
+    kind: GitRepository
+    name: storetheindex
+  wait: true
+  timeout: 2m
+  prune: false # TODO: Re-enable this once the team is used to GitOps.
```

Comment:
Why would this be false while we're getting used to the system?

Reply:
Setting it to true would revert any ad-hoc changes made directly via kubectl. I thought I make sure the team is comfortable with achieving what they would otherwise do via direct `kubectl *` and are not blocked by the new machinery before turning this on :)